### PR TITLE
Rtf 1414 workspace create strategies

### PR DESCRIPTION
**Pull request by maria-kizhaeva**
DRAFT 
Pr just to check if new API is working on the servers, my VM has some issues with it.

### CODE REVIEW

File: source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/ResourcePoolGetEligibleStrategy.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		public List<NamedArtifact> GetEligibleResourcePools()
+		{
+			var result = _restService.Get<List<JObject>>($"Relativity.Workspaces/workspace/eligible-resource-pools");
+			ConvertResult(result);
+			return new List<NamedArtifact>();
+		}
+
+		private void ConvertResult(List<JObject> result)
+		{
+			throw new NotImplementedException();
```
Comment (klaudiaknafel) [inline source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/ResourcePoolGetEligibleStrategy.cs:29]: ?

File: source/Relativity.Testing.Framework.Api/Strategies/Workspaces/WorkspaceCreateStrategyV1.cs [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+			return int.Parse(result[nameof(Artifact.ArtifactID)].ToString());
+		}
+
+		private int ResolveTemplateWorkspaceId(NamedArtifact templateWorkspace)
+		{
+			if (templateWorkspace == null)
+			{
+				var workspaces = _getAllWorkspacesStrategy.GetAll();
+
+				if (!workspaces.Any())
```
Comment (klaudiaknafel) [inline source/Relativity.Testing.Framework.Api/Strategies/Workspaces/WorkspaceCreateStrategyV1.cs:89]: can you extract this to separate method e.g. GetDefaultWorkspaceID?

File: source/Relativity.Testing.Framework.Api/Strategies/Workspaces/WorkspaceCreateStrategyV1.cs [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+			if (entity.SqlServer == null)
+			{
+				var resourcePool = _objectService.GetAll<ResourcePool>().FirstOrDefault(x => x.Name == entity.ResourcePool.Name);
+				entity.SqlServer = resourcePool.SqlServers.FirstOrDefault();
+			}
+
+			return new
+			{
+				WorkspaceRequest = new
```
Comment (klaudiaknafel) [inline source/Relativity.Testing.Framework.Api/Strategies/Workspaces/WorkspaceCreateStrategyV1.cs:157]: can this be extracted to separate method or moved to request dto with constructor?

File: source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/ResourcePoolGetEligibleStrategy.cs (hunk 1 above)
Comment (maria-kizhaeva) [reply source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/ResourcePoolGetEligibleStrategy.cs:29]: @klaudiaknafel it's a draft, not a valid PR :) see the desc. above. Anyway, I tested what I needed to test, I'm removing this PR

**Comment by maria-kizhaeva** [comment]
It was a draft pr just to be able run the tests against valid env-s while I was dealing with renewing TestVM

File: source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/ResourcePoolGetEligibleStrategy.cs (hunk 1 above)
Comment (briantoms) [reply source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/ResourcePoolGetEligibleStrategy.cs:29]: Make sure you select the Draft option in GitHub so it doesn't automatically add reviewers.

**Pull request closed without merging**